commons-app/apps-android-commons
# Fixes #3879 #4283  Other pictures flickers when a media is selected from contributions list

**Description (required)**
I am sorry for previous pull request as i panicked and push the wrong one

Fixes #3879 #4283

What changes did you make and why?
I just added a feature to check it the clicked item is loaded before setting that item, which will remove image flicker as program will not wait for 5 milliseconds and random crashes

**Tests performed (required)**
manual testing

please check this i am really sorry about previous pull request @nicolas-raoul @madhurgupta10 @neslihanturan
